# Perché ci serva la statistica?

In Fisica ci sono diversi problemi che richiedono l'uso della statistica. Quelli principali sono:
1. le misure, in particolare quelle di precisione
1. la verifica di nuove ipotesi. Tra queste, la più ambiziosa è la ricerca di nuovi fenomeni che possono portare a nuove scoperte

<img style="width: 500px" src="img/lep-zmass.png">

Misure dalle quali è stata determinata con precisione la massa della $Z$

<img style="width: 500px" src="img/cms-h2gg.png">
Spettro di massa di coppie di fotoni dal quale si è determinata evidenza del bosone di Higgs.

## Misure

I risultati di un esperimento sono spesso diverse l'una dall'altra, anche se l'esperimento viene ripetuto mantenendo le stesse condizioni. I motivi sono molteplici.
* Non si riesce a mantenere esattamente identiche le condizioni di lavoro di un esperimento. Ad es.: cambiano temperatura, umidità o altri parametri di lavoro.
* Gli strumenti di misura usati (i rivelatori) hanno fluttuazioni intrinseche nel loro funzionamento. La risposta è caratterizzata da una *risoluzione sperimentale* non ideale, e/o l'efficienza più essere non perfetta.
* La fisica stessa può avere casualità intrinseca: questo è inevitabile per fenomeni legati alla meccanica quantistica, ma anche fenomeni governati dalla fisica classica possono avere fortissime dipendenze da piccolissime variazioni dei parametri di lavoro (effetti caotici).

Anche se abbiamo *una singola misura*, per determinare una grandezza che ci interessa, spesso *non conosciamo* tutti gli effetti di cui dobbiamo tenere conto.

Cosideriamo, ad esempio, per la misura dell'accelerazione di gravità $g$, e immaginiamo di voler misurare il solo effetto dovuto alla gravità Terrestre.
Oltre ai numerosi effetti sperimentali che possono essere tenuti più o meno sotto controllo, la misura potrebbe dipendere dalla posizione in un particolare istante del Sole e della Luna, il cui contributo andrebbe sottratto al valore misurato per determinare il solo contributo a $g$ dovuto alla Terra. Se le posizioni di Sole e Luna non si conoscono, si può calcolare il livello di precisione che si può ottenere se si ignorano questi effetti. Il contributo dovuto ad una massa esterna vale:

$\displaystyle g^\prime = G\frac{M}{r^2}\ ,$

dove $G=6.67\times 10^{-11}{\mathrm{N}^2\mathrm{m}^2}/{\mathrm{kg}^2}$, $M$ e $r$ sono la massa e la distanza della massa (es.: Sole o Luna). Nel nostro caso:

$M_{\mathrm{Luna}} = 7.342\times 10^{22}\,\mathrm{kg},\ \ r_{\mathrm{Luna}} = 3.64 - 4.05\times 10^8\,\mathrm{m}$

$M_{\mathrm{Sole}} = 1.989\times 10^{30}\,\mathrm{kg},\ \ r_{\mathrm{Sole}} = 1.47-1.52 \times 10^{11}\,\mathrm{m}$

E quindi abbiamo i contributi:

In [3]:
g_newton = 6.67e-11
m_moon, r_moon = 7.342e22, 3.844e8
m_sun, r_sun = 1.989e30, 1.50e11
def g_prime(m, r): return g_newton * m / r**2
g_moon = g_prime(m_moon, r_moon)
g_sun = g_prime(m_sun, r_sun)
print('g\' moon = {:.6f} m/s²'.format(g_moon))
print('g\' sun = {:.6f} m/s²'.format(g_sun))

g' moon = 0.000033 m/s²
g' sun = 0.005896 m/s²


Confrontando con l'accelerazione sulla terra, $g=9.81\,\mathrm{m}/\mathrm{s}^2$, $g^{\,\prime}$ potrebbe sommarsi o sottrarsi, anche parzialmente, a seconda se le direzioni di $\vec{g}$ e $\vec{g}^{\,\prime}$ sono concordo o discordi, e in generale dell'angolo che formano, che non conosciamo. Quindi le variazioni relative entro le quali non è possibile misurare $g$ con ulteriore accuratezza sono:

In [3]:
g = 9.81
dg_rel = (g_sun + g_moon)/g
print('Δg/g = {:.3f}%'.format(dg_rel*100))

Δg/g = 0.060%


Questo effetto è piccolo, ma in realtà ci sono molti effetti, più o meno piccoli, non noti, o noti entro certi livelli di incertezza, i cui controbuti possono essere determinanti per la misura di interesse, a seconda del contesto sperimentale che abbiamo.

Detto questo, se conoscessimo la posizione di Terra e Luna, questo effetto si potrebbe correggere, introducendo, però, l'effetto dell'errore con il quale si conoscono queste due posizioni, che sarà molto più piccolo dello $0.060\%$ stimato sopra. Nella pratica, questo è irrilevante, a meno di non raggiungere i livelli di precisione comparabili con questo effetto.

Il livello di precisione raggiungibile, quindi, dipende dal metodo di misura e dal modo in cui si riesce a tener conto dei vari effetti e a correggere la misura per tenerne conto.

## Confronto con la teoria

Le distribuzioni delle quantità misurate sono previste da **modelli teorici** che in generale dipendono da **parametri**, come costanti più o meno fondamentali ($g$ nel caso prevedente), ecc.

Dato un campione di dati misurato vogliamo:
* misurare i parametri teorici
    * la massa di una particella
    * una costante di un'interazione fondamentale
    * ...ma anche l'efficienza o la risoluzione di uno strumento...
    * ...
* rispondere a domande sulla natura dei dati:
    * esiste il bosone di Higgs?
    * esistono le onde gravitazionali?
    * esiste la materia oscura?
    * se non abbiamo evidenza di una particella, quale intervallo della sua massa possiamo escludere?
    * ...
    
Per affrontare questi problemi dobbiamo:
* innanzitutto avere sotto controllo una **teoria della probabilità** che descriva la casualità intrinseca dei nostri dati, oppure gli effetti che influenzano la misura che *non conosciamo* (es.: l'effetto di Luna e Sole nella misura di $g$)
* applicare la teoria della probabilità al nostro campione di dati per rispondere alle nostre domande. Ad esempio:
    * la massa del quark top vale: $m_\mathrm{t} = 173.1 \pm 1.3\,\mathrm{GeV}$
    * il bosone di Higgs esiste; lo affermiamo perché possiamo definire una quantità (la **significatività statistica**) legata al campione osservato che misura con che probabilità questo potrebbe essere prodotto da una fluttuazione del fondo
    
Daremo significato alle affermazioni di cui sopra durante queste lezioni.

## Probabilità

Dobbiamo quindi definire cosa intendiamo per probabilità e capirne le *leggi*. Questo sarà lo scopo delle prime lezioni di questo corso.

Come prima cosa, capiremo cosa sono i fenomeni casuali e impareremo a simularli numericamente. In questo modo, avremo un laboratorio col quale esercitarci.

## Ma prima...

... ma prima sarà necessario prendere confidenza con alcuni strumenti software per seguire meglio queste lezioni e per svolgere le parti interative e di esercitazioni.